README.md

```diff
@@ -11,41 +11,51 @@ using [Syft](https://github.com/anchore/syft).
 
 By default, this action will execute a Syft scan in the workspace directory
 and upload a workflow artifact SBOM in SPDX format. It will also detect
-if being run during a `release` and attach the SBOM
-as a release asset.
+if being run during a [GitHub release](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases)
+and upload the SBOM as a release asset.
 
 ## Example Usage
 
-### Scan a docker image
+### Scan a container image
 
-Use the `image` parameter
+To scan a container image using the docker daemon use the `image` parameter:
 
 ```yaml
 - uses: anchore/sbom-action@main
   with:
     image: example/image_name
 ```
 
+With a container registry:
+
+```yaml
+- uses: anchore/sbom-action@main
+  with:
+    image: ghcr.io/example/image_name:tag
+```
+
 ### Scan a specific directory
 
-Use the `path` parameter, relative to the repository root
+Use the `path` parameter, relative to the repository root:
```

Comment:
A thought just occurred to me — are we validating the path to make sure it's at the repository root directory or a subdirectory therein? That is, not a parent directory or somewhere else on the system... I think we'd want the action to prevent that kind of misuse... what are your thoughts?

Reply:
I think we'd want to allow people to scan anywhere, they might put things in an explicit location like `/tmp/...`

Reply:
Ah — that's a good point!